### ISSUE TITLE
[Warrior] Rampage Passive stop working

### ISSUE THREAD

**Issue by AlexCSCP** [dalaranwow/dalaran-wow#5482]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Changing to a spec with rampage, being already affected by another warrior's rampage, make you lose your rampage.
**Expected behaviour**: Tell us what should happen instead
Rampage is a passive skill, it should always work.
**Steps to reproduce the problem**:

1. Join a party with a warrior having rampage.
2. Change spec to one with rampage.
3. Leave the party and see you don't have rampage.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
29801
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
https://i.imgur.com/YVVhlHe.jpg
As seen on the screenshot, I have no Rampage buff icon, and no +5% critical (current 10% = 5% for agility and 5% for cruelty).

**Comment by AlexCSCP** [comment]
This was mentioned in a comment on another issue, but that issue was closed without this problem being fixed.
(https://github.com/dalaranwow/dalaran-wow/issues/2353#issuecomment-228400254).
